src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,6 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return ((a ^ b) && (c ^ d)) || ((b ^ c) && (a ^ d));
```

Comment:
```suggestion
        return (a ^ b) && (c ^ d) || (b ^ c) && (a ^ d);
```
what are brackets for?

Reply:
Done